packages/codemirror/src/Editor.ts

```diff
@@ -32,6 +38,22 @@ export class Editor extends HTMLWidget {
         return this;
     }
 
+    markText(start: MarkTextPosition, end: MarkTextPosition): this {
+        if (this._codemirror) {
+            start.line--;
+            end.line--;
+            this._codemirror.markText(start, end, {className: "cm-marked-text"});
+        }
+        return this;
+    }
+
+    clearMarkedText(): this {
+        for (const marked of this._markedText) {
+            marked.clear();
+        }
```

Comment:
I doubt this works....

Reply:
It does work. I'll add an "app" to showcase it when make the requested rename changes.

Reply:
FWIW I don't see any code which adds new rows to this._markedText...

Reply:
*smack forehead* you're right, it was working on my test page because I was modifying the `markText` method on Editor's prototype